packages/story-editor/src/components/panels/design/videoOptions/videoOptions.js

```diff
@@ -159,14 +177,18 @@ function VideoOptionsPanel({ selectedElements, pushUpdate }) {
           </Text>
         </Label>
         {hasTrimMode && (
-          <TrimButton
-            variant={BUTTON_VARIANTS.RECTANGLE}
-            type={BUTTON_TYPES.SECONDARY}
-            size={BUTTON_SIZES.SMALL}
-            onClick={toggleTrimMode}
-          >
-            {__('Trim', 'web-stories')}
-          </TrimButton>
+          <TrimWrapper>
+            <TrimButton
+              disabled={isTrimming}
+              variant={BUTTON_VARIANTS.RECTANGLE}
+              type={BUTTON_TYPES.SECONDARY}
+              size={BUTTON_SIZES.SMALL}
+              onClick={toggleTrimMode}
+            >
+              {trimButtonText}
+            </TrimButton>
+            {isTrimming && <CircularProgress size={32} />}
```

Comment:
Added the circular progress here for making it more visible when trimming is actually happening. Let me know if this makes sense to you, then we could add it for Mute as well
<img width="236" alt="Screenshot 2021-09-24 at 13 36 45" src="https://user-images.githubusercontent.com/3294597/134662021-9786fc4b-6431-4007-91ae-5b7017974a68.png">

Reply:
Looks a bit odd, but I like the thought.

WDYT about just changing the text to "Trimming…" with ellipsis?

Reply:
You mean instead of using the spinner?

Could be, it's just that it's not very easily noticeable maybe. E.g. I realized the text of the Mute button was changing for the time of muting only when I saw it in code.

What if it's a bit smaller, e.g.

![Sep-24-2021 14-36-12](https://user-images.githubusercontent.com/3294597/134668138-61546aa2-1c3b-468a-bbbf-afea7483f0d0.gif)

If you still think it looks odd, let's use the `Trimming...` and we could bring it up with UX to follow up later for both buttons.

Reply:
> What if it's a bit smaller, e.g.

Yeah that helps 👍 

> If you still think it looks odd, let's use the Trimming...

Let's do both actually, while at it